lib/css/CssModulesPlugin.js

```diff
@@ -125,53 +126,51 @@ class CssModulesPlugin {
 					StaticExportsDependency,
 					new StaticExportsDependency.Template()
 				);
-				normalModuleFactory.hooks.createParser
-					.for("css")
-					.tap(plugin, parserOptions => {
-						validateParserOptions(parserOptions);
-						return new CssParser();
-					});
-				normalModuleFactory.hooks.createParser
-					.for("css/global")
-					.tap(plugin, parserOptions => {
-						validateParserOptions(parserOptions);
-						return new CssParser({
-							allowPseudoBlocks: false,
-							allowModeSwitch: false
+				for (const type of ["css", "css/global", "css/module"]) {
```

Comment:
Since you have already added `CSS` asset type in `ModuleTypeConstants` can you just add the other ones listed here.

Reply:
Yeah :+1: